src/async/tasks.md

```diff
@@ -60,4 +60,10 @@ Copy this example into your prepared `src/main.rs` and run it from there.
 
 * Refactor the async block into a function, and improve the error handling using `?`.
 
+* `nc` or `telnet` are good tools for this. It's not HTTPS
+
+     - [Netcat (nc)](https://docs.rs/nc/latest/nc/): Netcat is a          versatile networking utility that can be used for various networking tasks, including connecting to network services.
+
+     - [Telnet](https://docs.rs/telnet/latest/telnet/): Telnet is a network protocol used for remote terminal connections to devices over a network. It's a simple way to test network connectivity.
```

Comment:
I think both of these links are different from what the descriptions are referring to! But, going to look for links, there's not much! Maybe these links to the manpage are best:
 * https://www.unix.com/man-page/linux/1/nc/
 * https://www.unix.com/man-page/linux/1/telnet/

Reply:
Thanks for the links @djmitche 
i have updated the link in my latest commit [Updated links](https://github.com/google/comprehensive-rust/pull/1382/commits/652b5945f61556f9f873c74baa9b00babd2ca08b)